pkg/cmd/pulumi/convert.go

```diff
@@ -128,5 +131,9 @@ func newConvertCmd() *cobra.Command {
 		//nolint:lll
 		&outDir, "out", ".", "The output directory to write the convert project to")
 
+	cmd.PersistentFlags().BoolVar(
+		//nolint:lll
+		&installDeps, "install-dependencies", true, "Whether to install dependencies specified in the pulumi project")
```

Comment:
maybe consider --no-install-dependencies? since the flag should change behavior.

Reply:
this flag might imply to the user that dependencies are not installed by default, but they are.

Reply:
Yeh I feel this should of copied 'new' https://github.com/pulumi/pulumi/blob/master/pkg/cmd/pulumi/new.go#L483 and be 'generate-only'

Reply:
@dixler @Frassle agreed, will update 😃